polkadot/xcm/xcm-builder/src/lib.rs

```diff
@@ -34,8 +34,8 @@ pub use location_conversion::{
 	ChildParachainConvertsVia, DescribeAccountId32Terminal, DescribeAccountIdTerminal,
 	DescribeAccountKey20Terminal, DescribeAllTerminal, DescribeBodyTerminal, DescribeFamily,
```

Comment:
I guess, you are missing re-export for `DescribeTreasuryVoiceTerminal` here.y You wont be able to use it outside of xcm-builder without that.